### DISCUSSION TITLE
What's the difference between the generic and specific windows release?

### DISCUSSION THREAD

**Discussion by EvocativeOlash** [zen-browser/desktop#1255] [Q&A] [answered]
What's the difference between the generic and specific windows release?

**Comment by siliconmeadow** [comment]
I wonder the same about the linux generic and specific releases.

**Comment by siliconmeadow** [comment, marked as answer]
It looks like you go for the specific release if your processor is in this list: https://github.com/zen-browser/desktop/blob/main/docs/requirements.md

**Comment by cyrneko** [comment]
> What's the difference between the generic and specific windows release?

The "specific" release is compiled for newer x86 processors (see [this list](https://github.com/zen-browser/desktop/blob/main/docs/requirements.md)) only. This *may* have some performance benefits on supported systems.

The "Generic" release works on *pretty much* any x86 CPU.

This is the case for all supported x86 platforms.

**Reply by EvocativeOlash** [reply]
Ah ok that helps thanks.

Maybe they should call generic retro or really really old pc lol
